dockerfiles/packaging/Dockerfile

```diff
@@ -0,0 +1,44 @@
+ARG DOCKER_REGISTRY=docker.io
+
+FROM scratch as src
+COPY / / 
+
+FROM ${DOCKER_REGISTRY}/datadog/dd-trace-ci:php-7.3_centos-6 as php-7.3-base
+COPY --from=src / /build/
+WORKDIR /build/
```

Comment:
Feeling like a newbie docker user 😄 , what is the advantage of having this `php-7.3-base` target? and probably the same applies to `src` target?

Reply:
@labbati 
personal lazyness. 

Every sub artifact build like this:
```
FROM php-7.3-base as php-7.3
RUN bash dockerfiles/packaging/build.sh 7.3 20180731
```

Would have to look like this:
```
FROM ${DOCKER_REGISTRY}/datadog/dd-trace-ci:php-7.3_centos-6 as php-7.3
COPY --from=src / /build/
WORKDIR /build/
RUN bash dockerfiles/packaging/build.sh 7.3 20180731
```